src/types.ts

```diff
@@ -166,6 +166,27 @@ export interface ICourierMessagesGetResponse {
   }>;
 }
 
+export interface ICourierMessageCancelResponse {
```

Comment:
do we need `reason` in here?

Reply:
Josh, no since `reason` is not on the message but rather it's on the event log.